src/pygeom.c

```diff
@@ -407,6 +412,14 @@ char get_geom(GeometryObject *obj, GEOSGeometry **out) {
     }
 }
 
+/* Get a GEOSPreparedGeometry pointer from a GeometryObject
+This function does not check obj's type */
```

Comment:
It would be good to add a note here that this will return `NULL` if the geometry has not yet been prepared.

Also - `get_geom` has extra processing to handle cases where `obj` is `None`.  It seems like it is important to do the same here, right?

Reply:
Instead, I implemented `get_geom_with_prepared`. This because `get_geom_prepared` would always go after `get_geom`, better to combine them in one function (to be sure the checks are done).